tests/pubSub.cpp

```diff
@@ -94,12 +80,10 @@ BOOST_AUTO_TEST_CASE(publish_receive_zeroconf)
     if( !servus::Servus::isAvailable() || getenv("TRAVIS"))
         return;
 
-    zeq::URI uri = test::buildPublisherURI();
-    zeq::Publisher publisher( uri );
-    zeq::Subscriber noSubscriber( uri );
-
-    uri.addQuery( "subscribeSelf", "true" );
-    zeq::Subscriber subscriber( uri );
+    zeq::Publisher publisher( test::buildUniqueSession( ));
+    zeq::Subscriber noSubscriber( publisher.getSession( ));
+    zeq::detail::Sender::getUUID() = servus::make_UUID(); // different machine
```

Comment:
I was reading the documentation of Subscriber and I was wondering, "how did he test subscription in the same machine"?